documentation-site/pages/guides/theming.mdx

```diff
@@ -224,22 +224,47 @@ const overrides = {
   },
 };
 
-const theme = createTheme(lightThemePrimitives, overrides);
+const theme = createTheme(primitives, overrides);
 ```
 
 In this example, we first reassign the accent color on our theme primitives to hot pink. Then we use `overrides` to re-map the secondary button colors to use our hot pink accent colors rather than the default primary colors.
 
-Between `parameters` and `overrides`, you should have a fair amount of control over theme object creation. Ideally, you can use `createTheme` to quickly construct a Base Web compliant theme object.
+### `createDarkTheme`
+
+One of the main reasons to have a dynamic theme is to toggle your app between light and dark modes. `createTheme` will by default map `primitives` to theme colors in a way suited to light themes. If you want to create a dark theme you can instead use the `createDarkTheme` function.
+
+```js
+import {createDarkTheme} from 'baseui';
+
+const primitives = {
+  accent: '#F127E4', // hot pink
+  accent50: '#FDEDFC',
+  accent100: '#FCD3F9',
+  accent200: '#F89FF3',
+  accent300: '#F45AEA',
+  accent400: '#F127E4',
+  accent500: '#B71DAD',
+  accent600: '#901788',
+  accent700: '#600F5B',
+};
+
+const darkTheme = createDarkTheme(primitives);
```

Comment:
Should we include the second `overrides` arg here as an example?

Reply:
Added a second snippet to show `overrides` too